### PR TITLE
Add app icon 

### PR DESCRIPTION
**Pull request by KonradIT**
Added app icon.

### CODE REVIEW

**Comment by KonradIT** [comment]
> app/src/main/java/io/rebble/store/fragment/WatchfaceListFragment.java 

It gave me an error so I renamed the file, because the class was WatchfaceListFragment not WatchFaceListFragment

**Comment by KonradIT** [comment]
Yup, fixed!

File: app/build.gradle [hunk 1]
```diff
@@ -14,24 +14,16 @@ android {
...
     }
-    signingConfigs {
-        release {
-            storeFile file("../release.jks")
-            storePassword "pebble"
-            keyAlias "release"
-            keyPassword "pebble"
-        }
-    }
+
```
Comment (qichuan) [inline app/build.gradle]: Why remove the signing configs?

File: app/build.gradle [hunk 2]
```diff
@@ -14,24 +14,16 @@ android {
...
         release {
             minifyEnabled false
-            signingConfig signingConfigs.release
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
 }
 
-final SUPPORT_LIB_VERSION = '25.1.0'
+final SUPPORT_LIB_VERSION = '25.0.0'
```
Comment (qichuan) [inline app/build.gradle]: Why downgrade the support lib version?

File: app/build.gradle (hunk 1 above)
Comment (KonradIT) [inline app/build.gradle]: Should fix that now, it was because of a bug that didn't let me build the app.

**Comment by KonradIT** [comment]
Should be OK now.

**Pull request merged**